omod/src/main/webapp/patientDashboardForm.jsp

```diff
@@ -82,6 +82,19 @@
 		}
 		return false;
     }
+	
+	//hide the feature(concepts that are not defined either by property or value).
+	window.onload = function showOrHide() {
```

Comment:
Well done on this. Do you mind sharing a small vedio or screenshots for this fix.